client-mvp-04/src/sass/elements/_dropdown.scss

```diff
@@ -0,0 +1,57 @@
+@import '../variables';
+
+.dropdown-container {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  .dropdown-wrapper {
+    display: grid;
+    justify-content: center;
+    .dropdown {
+      display: flex;
+      justify-content: center;
+      width: 230px;
+      border-bottom: 1px solid $dropdown-border-color;
+      margin-bottom: 14px;
+      .selected {
+        width: 100%;
+        min-height: 20px;
+        text-align: left;
+        margin: 0px 0px 6px 3px;
+        line-height: 20px;
+        overflow-wrap: break-word;
+      }
+      .arrow {
+        width: 0;
+        height: 0;
+        margin: auto 14.8px 12px 0px;
+        border-left: 5.21px solid transparent;
+        border-right: 5.19px solid transparent;
+        border-top: 8.25px solid $dark-primary-color;
+      }
+    }
+    .option-container {
+      width: 232px;
```

Comment:
Now when select opens options it's moving a little bit.
I think it's because you have width: 230px for `dropdown `and for `option-container` 232px.
It needs to be consistent => 230px.

Reply:
Actually, in the Figma UI it was like the options container is `232`px and options are `230`px. If we make equal then this happens. Because we are using `box-sizing:border-box` globally. Padding is adding extra width.
![image](https://user-images.githubusercontent.com/43095489/118364339-e3c75200-b5b9-11eb-9ba7-e579d63c16a6.png)

Reply:
> Actually, in the Figma UI it was like this. Options container is `232`px and options are `230`px. If we make equal then this happens.

I resolved it by making **option-label** `width : 100%`